### DISCUSSION TITLE
Panzer Corps DLC not available

### DISCUSSION THREAD

**Discussion by mcv** [Heroic-Games-Launcher/HeroicGamesLauncher#3646] [General]
I bought Panzer Corps Gold (including all DLC) from GOG, and while the base game runs fine, the DLC is unavailable in the game. I can see the files sitting in ~/Games/Heroic/Panzer Corps/DLC, but the game clearly can't find it there. I've set the Wine prefix folder to that location, but that doesn't help.

There's also a Windows-like directory structure under pfx/drive_c, but I have no idea where I should put the DLC, if I have to put it there. I can't find anything online about this issue.

**Comment by imLinguin** [comment]
Check, `Modify Installation` found on the game page to see if all DLCs are selected. I'm not aware of any issues with this game either.

**Comment by abhortheabyss** [comment]
This is definitely not an issue of Heroic Games Launcher, but of the game itself. I'm in the same situation on normal Windows 10 and googling the problem brought me here. However I don't know the solution yet.
